app/views/fsss_view.py

```diff
@@ -53,6 +58,23 @@ def get_emp_info(name, on_false={}, on_missing='Unspecified'):
 
   preferred_name = get_emp_info('preferredName', on_false='None')
 
+  gsuite_filter = {
+      'rangeHigh': 10,
+      'rangeLow': 0,
+      'unique_employee_id': get_emp_info('uniqueEmployeeId'),
+  }
+
+  get_microservice_info = get_microservice_records("gsuitetable",
```

Comment:
Please use single-quotes for consistency, or run the formatter and see if the formatter fixes it.

Reply:
Adjusted, however the formatter did not fix this automatically